From 2a0ea486d1cf486ef069f6b4261ba53aa06541fd Mon Sep 17 00:00:00 2001
From: Orion Hodson <oth@google.com>
Date: Thu, 15 Dec 2022 10:04:22 +0000
Subject: [PATCH] Drop TrustCor certificates

INIT     Store initialized from system/ca-certificates/files
REMOVE   Android security review
         SHA1      : FF:BD:CD:E7:82:C8:43:5E:3C:6F:26:86:5C:CA:A8:3A:45:5B:C3:0A
         Subject   : C=PA, ST=Panama, L=Panama City, O=TrustCor Systems S. de R.L., OU=TrustCor Certificate Authority, CN=TrustCor RootCert CA-1
         Not Before: Feb  4 12:32:16 2016 GMT
         Not After : Dec 31 17:23:16 2029 GMT
REMOVE   Android security review
         SHA1      : B8:BE:6D:CB:56:F1:55:B9:63:D4:12:CA:4E:06:34:C7:94:B2:1C:C0
         Subject   : C=PA, ST=Panama, L=Panama City, O=TrustCor Systems S. de R.L., OU=TrustCor Certificate Authority, CN=TrustCor RootCert CA-2
         Not Before: Feb  4 12:32:23 2016 GMT
         Not After : Dec 31 17:26:39 2034 GMT
REMOVE   Android security review
         SHA1      : 58:D1:DF:95:95:67:6B:63:C0:F0:5B:1C:17:4D:8B:84:0B:C8:78:BD
         Subject   : C=PA, ST=Panama, L=Panama City, O=TrustCor Systems S. de R.L., OU=TrustCor Certificate Authority, CN=TrustCor ECA-1
         Not Before: Feb  4 12:32:33 2016 GMT
         Not After : Dec 31 17:28:07 2029 GMT
IMPORT   Starting import from vendor/google/tools/cacerts/certdata.txt
PROCESS  Handling 158 cert requests
EXPIRING Cert is expiring on May 15 04:52:29 2023 GMT
         SHA1      : D6:DA:A8:20:8D:09:D2:15:4D:24:B5:2F:CB:34:6E:B2:58:B2:8A:58
         Subject   : C=HK, O=Hongkong Post, CN=Hongkong Post Root CA 1
         Not Before: May 15 05:13:14 2003 GMT
         Not After : May 15 04:52:29 2023 GMT
EXPIRING Cert is expiring on Mar  3 12:09:48 2023 GMT
         SHA1      : 51:C6:E7:08:49:06:6E:F3:92:D4:5C:A0:0D:6D:A3:62:8F:C3:52:39
         Subject   : C=TR, L=Ankara, O=E-Tu\xC4\x9Fra EBG Bili\xC5\x9Fim Teknolojileri ve Hizmetleri A.\xC5\x9E., OU=E-Tugra Sertifikasyon Merkezi, CN=E-Tugra Certification Authority
         Not Before: Mar  5 12:09:48 2013 GMT
         Not After : Mar  3 12:09:48 2023 GMT
EXPIRING Cert is expiring on Sep 30 04:20:49 2023 GMT
         SHA1      : 36:B1:2B:49:F9:81:9E:D7:4C:9E:BC:38:0F:C6:56:8F:5D:AC:B2:F7
         Subject   : C=JP, O=SECOM Trust.net, OU=Security Communication RootCA1
         Not Before: Sep 30 04:20:49 2003 GMT
         Not After : Sep 30 04:20:49 2023 GMT

SUMMARY

  CertRequest
                 total count: 158
                   add count:  30
                remove count: 124
                 trust count:   0

  Repository
               total trusted: 136
            roots for import: 137
     intermediate for import:   0

  Store (before import)
                 total count: 140
                     expired:   0
        expiring with 1 year:   3

  Store (after import)
                 total count: 137
              newly imported:   0
  existing matched by import: 137
               other trusted:   0
                     expired:   0
        expiring with 1 year:   3

  Store (after cert requests)
                 total count: 137
              newly imported:   0
  existing matched by import: 137
               other trusted:   0
                     expired:   0
        expiring with 1 year:   3

Bug: 262521447
Test: atest CtsLibcoreTestCases:com.android.org.conscrypt.TrustedCertificateStoreTest

(cherry picked from commit 394a6486d9e6aa9c4a0ebef3fe3c4f5ede63b50f)
(cherry picked from https://android-review.googlesource.com/q/commit:fb5214798155bf7e5a14bf286a0f3725e69b23d1)
(cherry picked from https://googleplex-android-review.googlesource.com/q/commit:cbc6bbaa091a1217001817023e8859315b8ecc33)
Merged-In: Id951f8710aff9f9a907f4fc56035533eb6f6b760
Change-Id: Id951f8710aff9f9a907f4fc56035533eb6f6b760
---
 files/7c302982.0 |  86 --------------------------------
 files/c2c1704e.0 |  86 --------------------------------
 files/d0cddf45.0 | 127 -----------------------------------------------
 3 files changed, 299 deletions(-)
 delete mode 100644 files/7c302982.0
 delete mode 100644 files/c2c1704e.0
 delete mode 100644 files/d0cddf45.0

diff --git a/files/7c302982.0 b/files/7c302982.0
deleted file mode 100644
index 51c8ed3..0000000
--- a/files/7c302982.0
+++ /dev/null
@@ -1,86 +0,0 @@
------BEGIN CERTIFICATE-----
-MIIEMDCCAxigAwIBAgIJANqb7HHzA7AZMA0GCSqGSIb3DQEBCwUAMIGkMQswCQYD
-VQQGEwJQQTEPMA0GA1UECAwGUGFuYW1hMRQwEgYDVQQHDAtQYW5hbWEgQ2l0eTEk
-MCIGA1UECgwbVHJ1c3RDb3IgU3lzdGVtcyBTLiBkZSBSLkwuMScwJQYDVQQLDB5U
-cnVzdENvciBDZXJ0aWZpY2F0ZSBBdXRob3JpdHkxHzAdBgNVBAMMFlRydXN0Q29y
-IFJvb3RDZXJ0IENBLTEwHhcNMTYwMjA0MTIzMjE2WhcNMjkxMjMxMTcyMzE2WjCB
-pDELMAkGA1UEBhMCUEExDzANBgNVBAgMBlBhbmFtYTEUMBIGA1UEBwwLUGFuYW1h
-IENpdHkxJDAiBgNVBAoMG1RydXN0Q29yIFN5c3RlbXMgUy4gZGUgUi5MLjEnMCUG
-A1UECwweVHJ1c3RDb3IgQ2VydGlmaWNhdGUgQXV0aG9yaXR5MR8wHQYDVQQDDBZU
-cnVzdENvciBSb290Q2VydCBDQS0xMIIBIjANBgkqhkiG9w0BAQEFAAOCAQ8AMIIB
-CgKCAQEAv463leLCJhJrMxnHQFgKq1mqjQCj/IDHUHuO1CAmujIS2CNUSSUQIpid
-RtLByZ5OGy4sDjjzGiVoHKZaBeYei0i/mJZ0PmnK6bV4pQa81QBeCQryJ3pS/C3V
-seq0iWEk8xoT26nPUu0MJLq5nux+AHT6k61sKZKuUbS701e/s/OojZz0JEsq1pme
-9J7+wH5COucLlVPat2gOkEz7cD+PSiyU8ybdY2mplNgQTsVHCJCZGxdNuWxu72CV
-EY4hgLW9oHPY0LJ3xEXqWib7ZnZ2+AYfYW0PVcWDtxBWcgYHpfOxGgMFZA6dWorW
-hnAbJN7+KIor0Gqw/Hqi3LJ5DotlDwIDAQABo2MwYTAdBgNVHQ4EFgQU7mtJPHo/
-DeOxCbeKyKsZn3MzUOcwHwYDVR0jBBgwFoAU7mtJPHo/DeOxCbeKyKsZn3MzUOcw
-DwYDVR0TAQH/BAUwAwEB/zAOBgNVHQ8BAf8EBAMCAYYwDQYJKoZIhvcNAQELBQAD
-ggEBACUY1JGPE+6PHh0RU9otRCkZoB5rMZ5NDp6tPVxBb5UrJKF5mDo4Nvu7Zp5I
-/5CQ7z3UuJu0h3U/IJvOcs+hVcFNZKIZBqEHMwwLKeXx6quj7LUKdJDHfXLy11yf
-ke+Ri7fc7Waiz45mO7yfOgLgJ90WmMCV1Aqk5IGadZQ1nJBfiDcGrVmVCrDRZ9MZ
-yonnMlo2HD6CqFqTvsbQZJG2z9m2GM/bftJlo6bEjhcxwft+dtvTheNYsnd6djts
-L1Ac59v2Z3kf9YKVmgenFK+P3CghZwnS1k1aHBkcjndcw5QkPTJrS37UeJSDvjdN
-zl/HHk484IkzlQsPpTLWPFp5LBk=
------END CERTIFICATE-----
-Certificate:
-    Data:
-        Version: 3 (0x2)
-        Serial Number:
-            da:9b:ec:71:f3:03:b0:19
-        Signature Algorithm: sha256WithRSAEncryption
-        Issuer: C=PA, ST=Panama, L=Panama City, O=TrustCor Systems S. de R.L., OU=TrustCor Certificate Authority, CN=TrustCor RootCert CA-1
-        Validity
-            Not Before: Feb  4 12:32:16 2016 GMT
-            Not After : Dec 31 17:23:16 2029 GMT
-        Subject: C=PA, ST=Panama, L=Panama City, O=TrustCor Systems S. de R.L., OU=TrustCor Certificate Authority, CN=TrustCor RootCert CA-1
-        Subject Public Key Info:
-            Public Key Algorithm: rsaEncryption
-                RSA Public-Key: (2048 bit)
-                Modulus:
-                    00:bf:8e:b7:95:e2:c2:26:12:6b:33:19:c7:40:58:
-                    0a:ab:59:aa:8d:00:a3:fc:80:c7:50:7b:8e:d4:20:
-                    26:ba:32:12:d8:23:54:49:25:10:22:98:9d:46:d2:
-                    c1:c9:9e:4e:1b:2e:2c:0e:38:f3:1a:25:68:1c:a6:
-                    5a:05:e6:1e:8b:48:bf:98:96:74:3e:69:ca:e9:b5:
-                    78:a5:06:bc:d5:00:5e:09:0a:f2:27:7a:52:fc:2d:
-                    d5:b1:ea:b4:89:61:24:f3:1a:13:db:a9:cf:52:ed:
-                    0c:24:ba:b9:9e:ec:7e:00:74:fa:93:ad:6c:29:92:
-                    ae:51:b4:bb:d3:57:bf:b3:f3:a8:8d:9c:f4:24:4b:
-                    2a:d6:99:9e:f4:9e:fe:c0:7e:42:3a:e7:0b:95:53:
-                    da:b7:68:0e:90:4c:fb:70:3f:8f:4a:2c:94:f3:26:
-                    dd:63:69:a9:94:d8:10:4e:c5:47:08:90:99:1b:17:
-                    4d:b9:6c:6e:ef:60:95:11:8e:21:80:b5:bd:a0:73:
-                    d8:d0:b2:77:c4:45:ea:5a:26:fb:66:76:76:f8:06:
-                    1f:61:6d:0f:55:c5:83:b7:10:56:72:06:07:a5:f3:
-                    b1:1a:03:05:64:0e:9d:5a:8a:d6:86:70:1b:24:de:
-                    fe:28:8a:2b:d0:6a:b0:fc:7a:a2:dc:b2:79:0e:8b:
-                    65:0f
-                Exponent: 65537 (0x10001)
-        X509v3 extensions:
-            X509v3 Subject Key Identifier: 
-                EE:6B:49:3C:7A:3F:0D:E3:B1:09:B7:8A:C8:AB:19:9F:73:33:50:E7
-            X509v3 Authority Key Identifier: 
-                keyid:EE:6B:49:3C:7A:3F:0D:E3:B1:09:B7:8A:C8:AB:19:9F:73:33:50:E7
-
-            X509v3 Basic Constraints: critical
-                CA:TRUE
-            X509v3 Key Usage: critical
-                Digital Signature, Certificate Sign, CRL Sign
-    Signature Algorithm: sha256WithRSAEncryption
-         25:18:d4:91:8f:13:ee:8f:1e:1d:11:53:da:2d:44:29:19:a0:
-         1e:6b:31:9e:4d:0e:9e:ad:3d:5c:41:6f:95:2b:24:a1:79:98:
-         3a:38:36:fb:bb:66:9e:48:ff:90:90:ef:3d:d4:b8:9b:b4:87:
-         75:3f:20:9b:ce:72:cf:a1:55:c1:4d:64:a2:19:06:a1:07:33:
-         0c:0b:29:e5:f1:ea:ab:a3:ec:b5:0a:74:90:c7:7d:72:f2:d7:
-         5c:9f:91:ef:91:8b:b7:dc:ed:66:a2:cf:8e:66:3b:bc:9f:3a:
-         02:e0:27:dd:16:98:c0:95:d4:0a:a4:e4:81:9a:75:94:35:9c:
-         90:5f:88:37:06:ad:59:95:0a:b0:d1:67:d3:19:ca:89:e7:32:
-         5a:36:1c:3e:82:a8:5a:93:be:c6:d0:64:91:b6:cf:d9:b6:18:
-         cf:db:7e:d2:65:a3:a6:c4:8e:17:31:c1:fb:7e:76:db:d3:85:
-         e3:58:b2:77:7a:76:3b:6c:2f:50:1c:e7:db:f6:67:79:1f:f5:
-         82:95:9a:07:a7:14:af:8f:dc:28:21:67:09:d2:d6:4d:5a:1c:
-         19:1c:8e:77:5c:c3:94:24:3d:32:6b:4b:7e:d4:78:94:83:be:
-         37:4d:ce:5f:c7:1e:4e:3c:e0:89:33:95:0b:0f:a5:32:d6:3c:
-         5a:79:2c:19
-SHA1 Fingerprint=FF:BD:CD:E7:82:C8:43:5E:3C:6F:26:86:5C:CA:A8:3A:45:5B:C3:0A
diff --git a/files/c2c1704e.0 b/files/c2c1704e.0
deleted file mode 100644
index 50d9e26..0000000
--- a/files/c2c1704e.0
+++ /dev/null
@@ -1,86 +0,0 @@
------BEGIN CERTIFICATE-----
-MIIEIDCCAwigAwIBAgIJAISCLF8cYtBAMA0GCSqGSIb3DQEBCwUAMIGcMQswCQYD
-VQQGEwJQQTEPMA0GA1UECAwGUGFuYW1hMRQwEgYDVQQHDAtQYW5hbWEgQ2l0eTEk
-MCIGA1UECgwbVHJ1c3RDb3IgU3lzdGVtcyBTLiBkZSBSLkwuMScwJQYDVQQLDB5U
-cnVzdENvciBDZXJ0aWZpY2F0ZSBBdXRob3JpdHkxFzAVBgNVBAMMDlRydXN0Q29y
-IEVDQS0xMB4XDTE2MDIwNDEyMzIzM1oXDTI5MTIzMTE3MjgwN1owgZwxCzAJBgNV
-BAYTAlBBMQ8wDQYDVQQIDAZQYW5hbWExFDASBgNVBAcMC1BhbmFtYSBDaXR5MSQw
-IgYDVQQKDBtUcnVzdENvciBTeXN0ZW1zIFMuIGRlIFIuTC4xJzAlBgNVBAsMHlRy
-dXN0Q29yIENlcnRpZmljYXRlIEF1dGhvcml0eTEXMBUGA1UEAwwOVHJ1c3RDb3Ig
-RUNBLTEwggEiMA0GCSqGSIb3DQEBAQUAA4IBDwAwggEKAoIBAQDPj+ARtZ+odnbb
-3w9U73NjKYKtR8aja+3+XzP4Q1HpGjORMRegdMTUpwHmspI+ap3tDvl0mEDTPwOA
-BoJA6LHip1GnHYMma6ve+heRK9jGrB6xnhkB1Zem6g23xFUfJ3zSCNV2HykVh0A5
-3ThFEXXQmqc04L/NyFIduUd+Dbi7xgz2c1cWWn5DkR9VOsZtRASqnKmcp0yJF4Ou
-owReUoCLHhIlERnXDH19MURB6tuvsBzvgdAsxZohmz3tQjtQJvLsznFhBmIhVE5/
-wZ0+fyCMgMsq2JdiyIMzkX2woloPV+g7zPIlstR8L+xNxqE6FXrntl019fZISjZF
-ZtS6mFjBAgMBAAGjYzBhMB0GA1UdDgQWBBREnkj1zG1I1KBLf/5ZJC+Dl5mahjAf
-BgNVHSMEGDAWgBREnkj1zG1I1KBLf/5ZJC+Dl5mahjAPBgNVHRMBAf8EBTADAQH/
-MA4GA1UdDwEB/wQEAwIBhjANBgkqhkiG9w0BAQsFAAOCAQEABT41XBVwm8nHc2Fv
-civUwo/yQ10CzsSUuZQRg2dd4mdsdXa/uwyqNsatR5Nj3B5+1t4u/ukZMjgDfxT2
-AHMsWbEhBuH7rBiVDKP/mZb3Kyeb1STMHd3BOuCYRLDE5D53sXOpZCz2HAF8P11F
-hcCF5yWPldwX8zyfGm6wyuMdKulMY/okYWLW2n62HGz1Ah3UKt1VkOsqEUc8Ll50
-soIipX1TH0XsJ5F95yIW6MBoNtjG8U+ARDL54dHRHareqKucBK+tIA5kmE2la8BI
-WJZpTdwHjFGTot+fDz2LYLSCjaoITmJF4PkL0uDgPFveXHEnJcLmA4GLEFPjx1Wi
-tJ/X5g==
------END CERTIFICATE-----
-Certificate:
-    Data:
-        Version: 3 (0x2)
-        Serial Number:
-            84:82:2c:5f:1c:62:d0:40
-        Signature Algorithm: sha256WithRSAEncryption
-        Issuer: C=PA, ST=Panama, L=Panama City, O=TrustCor Systems S. de R.L., OU=TrustCor Certificate Authority, CN=TrustCor ECA-1
-        Validity
-            Not Before: Feb  4 12:32:33 2016 GMT
-            Not After : Dec 31 17:28:07 2029 GMT
-        Subject: C=PA, ST=Panama, L=Panama City, O=TrustCor Systems S. de R.L., OU=TrustCor Certificate Authority, CN=TrustCor ECA-1
-        Subject Public Key Info:
-            Public Key Algorithm: rsaEncryption
-                RSA Public-Key: (2048 bit)
-                Modulus:
-                    00:cf:8f:e0:11:b5:9f:a8:76:76:db:df:0f:54:ef:
-                    73:63:29:82:ad:47:c6:a3:6b:ed:fe:5f:33:f8:43:
-                    51:e9:1a:33:91:31:17:a0:74:c4:d4:a7:01:e6:b2:
-                    92:3e:6a:9d:ed:0e:f9:74:98:40:d3:3f:03:80:06:
-                    82:40:e8:b1:e2:a7:51:a7:1d:83:26:6b:ab:de:fa:
-                    17:91:2b:d8:c6:ac:1e:b1:9e:19:01:d5:97:a6:ea:
-                    0d:b7:c4:55:1f:27:7c:d2:08:d5:76:1f:29:15:87:
-                    40:39:dd:38:45:11:75:d0:9a:a7:34:e0:bf:cd:c8:
-                    52:1d:b9:47:7e:0d:b8:bb:c6:0c:f6:73:57:16:5a:
-                    7e:43:91:1f:55:3a:c6:6d:44:04:aa:9c:a9:9c:a7:
-                    4c:89:17:83:ae:a3:04:5e:52:80:8b:1e:12:25:11:
-                    19:d7:0c:7d:7d:31:44:41:ea:db:af:b0:1c:ef:81:
-                    d0:2c:c5:9a:21:9b:3d:ed:42:3b:50:26:f2:ec:ce:
-                    71:61:06:62:21:54:4e:7f:c1:9d:3e:7f:20:8c:80:
-                    cb:2a:d8:97:62:c8:83:33:91:7d:b0:a2:5a:0f:57:
-                    e8:3b:cc:f2:25:b2:d4:7c:2f:ec:4d:c6:a1:3a:15:
-                    7a:e7:b6:5d:35:f5:f6:48:4a:36:45:66:d4:ba:98:
-                    58:c1
-                Exponent: 65537 (0x10001)
-        X509v3 extensions:
-            X509v3 Subject Key Identifier: 
-                44:9E:48:F5:CC:6D:48:D4:A0:4B:7F:FE:59:24:2F:83:97:99:9A:86
-            X509v3 Authority Key Identifier: 
-                keyid:44:9E:48:F5:CC:6D:48:D4:A0:4B:7F:FE:59:24:2F:83:97:99:9A:86
-
-            X509v3 Basic Constraints: critical
-                CA:TRUE
-            X509v3 Key Usage: critical
-                Digital Signature, Certificate Sign, CRL Sign
-    Signature Algorithm: sha256WithRSAEncryption
-         05:3e:35:5c:15:70:9b:c9:c7:73:61:6f:72:2b:d4:c2:8f:f2:
-         43:5d:02:ce:c4:94:b9:94:11:83:67:5d:e2:67:6c:75:76:bf:
-         bb:0c:aa:36:c6:ad:47:93:63:dc:1e:7e:d6:de:2e:fe:e9:19:
-         32:38:03:7f:14:f6:00:73:2c:59:b1:21:06:e1:fb:ac:18:95:
-         0c:a3:ff:99:96:f7:2b:27:9b:d5:24:cc:1d:dd:c1:3a:e0:98:
-         44:b0:c4:e4:3e:77:b1:73:a9:64:2c:f6:1c:01:7c:3f:5d:45:
-         85:c0:85:e7:25:8f:95:dc:17:f3:3c:9f:1a:6e:b0:ca:e3:1d:
-         2a:e9:4c:63:fa:24:61:62:d6:da:7e:b6:1c:6c:f5:02:1d:d4:
-         2a:dd:55:90:eb:2a:11:47:3c:2e:5e:74:b2:82:22:a5:7d:53:
-         1f:45:ec:27:91:7d:e7:22:16:e8:c0:68:36:d8:c6:f1:4f:80:
-         44:32:f9:e1:d1:d1:1d:aa:de:a8:ab:9c:04:af:ad:20:0e:64:
-         98:4d:a5:6b:c0:48:58:96:69:4d:dc:07:8c:51:93:a2:df:9f:
-         0f:3d:8b:60:b4:82:8d:aa:08:4e:62:45:e0:f9:0b:d2:e0:e0:
-         3c:5b:de:5c:71:27:25:c2:e6:03:81:8b:10:53:e3:c7:55:a2:
-         b4:9f:d7:e6
-SHA1 Fingerprint=58:D1:DF:95:95:67:6B:63:C0:F0:5B:1C:17:4D:8B:84:0B:C8:78:BD
diff --git a/files/d0cddf45.0 b/files/d0cddf45.0
deleted file mode 100644
index e41f666..0000000
--- a/files/d0cddf45.0
+++ /dev/null
@@ -1,127 +0,0 @@
------BEGIN CERTIFICATE-----
-MIIGLzCCBBegAwIBAgIIJaHfyjPLWQIwDQYJKoZIhvcNAQELBQAwgaQxCzAJBgNV
-BAYTAlBBMQ8wDQYDVQQIDAZQYW5hbWExFDASBgNVBAcMC1BhbmFtYSBDaXR5MSQw
-IgYDVQQKDBtUcnVzdENvciBTeXN0ZW1zIFMuIGRlIFIuTC4xJzAlBgNVBAsMHlRy
-dXN0Q29yIENlcnRpZmljYXRlIEF1dGhvcml0eTEfMB0GA1UEAwwWVHJ1c3RDb3Ig
-Um9vdENlcnQgQ0EtMjAeFw0xNjAyMDQxMjMyMjNaFw0zNDEyMzExNzI2MzlaMIGk
-MQswCQYDVQQGEwJQQTEPMA0GA1UECAwGUGFuYW1hMRQwEgYDVQQHDAtQYW5hbWEg
-Q2l0eTEkMCIGA1UECgwbVHJ1c3RDb3IgU3lzdGVtcyBTLiBkZSBSLkwuMScwJQYD
-VQQLDB5UcnVzdENvciBDZXJ0aWZpY2F0ZSBBdXRob3JpdHkxHzAdBgNVBAMMFlRy
-dXN0Q29yIFJvb3RDZXJ0IENBLTIwggIiMA0GCSqGSIb3DQEBAQUAA4ICDwAwggIK
-AoICAQCnIG7CKqJiJJWQdsg4foDSq8GbZQWU9MEKENUCrO2fk8eHyLAnK0IMPQo+
-QVqedd2NyuCb7GgypGmSaIwLgQ5WoD4a3SwlFIIvl9NkRvRUqdw6VC0xK5mC8tkq
-1+9xALgxpL56JAfDQiDyitSSBBtlVkxs1Pu2YVpHI7TYabS3OtB0PAx1oYxOdqHp
-2yqlO/rOsP9+aij9JxzIsekp8VduZLTQwRVtDr4uDkbIXvRR/u8OYzo7cbrPb1nK
-DOObXUm4TOJXsZiKQlecdu/vvdFoqNL0Cbt3Nb4lggjEFixEIFapRBF37120Hape
-az6LMvYHL1cEksr1/p3C6eizjkxLAjHZ5DxIgif3GIJ2SDpxsROhOdUuxTTCHWKF
-3wP+TfSvPd9cW436cOGlfifHhi5qjxLGhF5DUVCcGZt45vz27Ud+ez1m7xMTiF88
-oWP7+ayHNZ/zgp6kPwqcMWmLmaSISo5uZk3vFsQPeSghYA2FFn3XVDjxklb9tTNM
-g9zXEJ9L/cb4Qr26fHMC4P99zVvh1Kxhe1fVSntb1IVYJ12/+CtgrKAmrhQhJ8Z3
-mjOAPF5GP/fDsaOGM8boXg25NSyqRsGFAnWAoOsk+xWq5Gd/bnc/9ASKL3x74xdh
-8N0JqSDIvgmk0H5Ew7IwSjiqqewYmgeCK9u4nBit2uBGF6zPXQIDAQABo2MwYTAd
-BgNVHQ4EFgQU2f4hQG6UnrybPZx9mCAZ5YwwYrIwHwYDVR0jBBgwFoAU2f4hQG6U
-nrybPZx9mCAZ5YwwYrIwDwYDVR0TAQH/BAUwAwEB/zAOBgNVHQ8BAf8EBAMCAYYw
-DQYJKoZIhvcNAQELBQADggIBAJ5Fngw7tu/hOsh80QA9z+LqBrWyOrsGS2h60COX
-dKcs8AjYeVrXWoSK2BKaG9l9XE1wxaX5q+WjiYndAfrs3fnpkpfbsEZC89NiqpX+
-MWcUaViQCqoL7jcjx1BRtPV+nuN79+TMQjItSQzL/0kMmx40/W5ulop5A7Zv2wnL
-/V9lFDfhOPXzYRZY5LVtDQsEGz9QLX+zx3oaFoBg+Iof6Rsqxvm6ARppv9JYx1RX
-CI/hOWB3S6xZhBqI8d3LT3jX5+EzLfzuQfogsL7L9ziUwOHQhQ+77Sxzq+3+knYa
-ZH9bDTMJBzN7Bj8RpFxwPIXAz+OQqIN3+tvmxYxoZxBnpVIt8MSZj3+/0WvitUfW
-2dCFmU2Umw9Lje4AWkcdEQOsQRivh7dvDDqPys/cA8GiCcjl/YBeyGBCARsaU1q7
-N6a3vLqE6R5sGtRk2tRD/pOLS/IseRYQ1JMLiI+h2IYURpFHmygk71dSTlxCnKr3
-Sewn6EAes6aJInKc9Q0ztFijMDvd1GpUk74aTfOTlPf8hAs/hCBcNANExdqtvArB
-As8e5ZTZ845b2EzwnexhF7sUMlQMAimTHpKG9n/v55IFDlndmQguLvqcAFLTxWYp
-5KeXRKQOKIETNcX2b2TmQcTVL8w0RSXPQQCWPUouwpaYT05KnJe32x+SMsj/D1Fu
-1uwJ
------END CERTIFICATE-----
-Certificate:
-    Data:
-        Version: 3 (0x2)
-        Serial Number: 2711694510199101698 (0x25a1dfca33cb5902)
-        Signature Algorithm: sha256WithRSAEncryption
-        Issuer: C=PA, ST=Panama, L=Panama City, O=TrustCor Systems S. de R.L., OU=TrustCor Certificate Authority, CN=TrustCor RootCert CA-2
-        Validity
-            Not Before: Feb  4 12:32:23 2016 GMT
-            Not After : Dec 31 17:26:39 2034 GMT
-        Subject: C=PA, ST=Panama, L=Panama City, O=TrustCor Systems S. de R.L., OU=TrustCor Certificate Authority, CN=TrustCor RootCert CA-2
-        Subject Public Key Info:
-            Public Key Algorithm: rsaEncryption
-                RSA Public-Key: (4096 bit)
-                Modulus:
-                    00:a7:20:6e:c2:2a:a2:62:24:95:90:76:c8:38:7e:
-                    80:d2:ab:c1:9b:65:05:94:f4:c1:0a:10:d5:02:ac:
-                    ed:9f:93:c7:87:c8:b0:27:2b:42:0c:3d:0a:3e:41:
-                    5a:9e:75:dd:8d:ca:e0:9b:ec:68:32:a4:69:92:68:
-                    8c:0b:81:0e:56:a0:3e:1a:dd:2c:25:14:82:2f:97:
-                    d3:64:46:f4:54:a9:dc:3a:54:2d:31:2b:99:82:f2:
-                    d9:2a:d7:ef:71:00:b8:31:a4:be:7a:24:07:c3:42:
-                    20:f2:8a:d4:92:04:1b:65:56:4c:6c:d4:fb:b6:61:
-                    5a:47:23:b4:d8:69:b4:b7:3a:d0:74:3c:0c:75:a1:
-                    8c:4e:76:a1:e9:db:2a:a5:3b:fa:ce:b0:ff:7e:6a:
-                    28:fd:27:1c:c8:b1:e9:29:f1:57:6e:64:b4:d0:c1:
-                    15:6d:0e:be:2e:0e:46:c8:5e:f4:51:fe:ef:0e:63:
-                    3a:3b:71:ba:cf:6f:59:ca:0c:e3:9b:5d:49:b8:4c:
-                    e2:57:b1:98:8a:42:57:9c:76:ef:ef:bd:d1:68:a8:
-                    d2:f4:09:bb:77:35:be:25:82:08:c4:16:2c:44:20:
-                    56:a9:44:11:77:ef:5d:b4:1d:aa:5e:6b:3e:8b:32:
-                    f6:07:2f:57:04:92:ca:f5:fe:9d:c2:e9:e8:b3:8e:
-                    4c:4b:02:31:d9:e4:3c:48:82:27:f7:18:82:76:48:
-                    3a:71:b1:13:a1:39:d5:2e:c5:34:c2:1d:62:85:df:
-                    03:fe:4d:f4:af:3d:df:5c:5b:8d:fa:70:e1:a5:7e:
-                    27:c7:86:2e:6a:8f:12:c6:84:5e:43:51:50:9c:19:
-                    9b:78:e6:fc:f6:ed:47:7e:7b:3d:66:ef:13:13:88:
-                    5f:3c:a1:63:fb:f9:ac:87:35:9f:f3:82:9e:a4:3f:
-                    0a:9c:31:69:8b:99:a4:88:4a:8e:6e:66:4d:ef:16:
-                    c4:0f:79:28:21:60:0d:85:16:7d:d7:54:38:f1:92:
-                    56:fd:b5:33:4c:83:dc:d7:10:9f:4b:fd:c6:f8:42:
-                    bd:ba:7c:73:02:e0:ff:7d:cd:5b:e1:d4:ac:61:7b:
-                    57:d5:4a:7b:5b:d4:85:58:27:5d:bf:f8:2b:60:ac:
-                    a0:26:ae:14:21:27:c6:77:9a:33:80:3c:5e:46:3f:
-                    f7:c3:b1:a3:86:33:c6:e8:5e:0d:b9:35:2c:aa:46:
-                    c1:85:02:75:80:a0:eb:24:fb:15:aa:e4:67:7f:6e:
-                    77:3f:f4:04:8a:2f:7c:7b:e3:17:61:f0:dd:09:a9:
-                    20:c8:be:09:a4:d0:7e:44:c3:b2:30:4a:38:aa:a9:
-                    ec:18:9a:07:82:2b:db:b8:9c:18:ad:da:e0:46:17:
-                    ac:cf:5d
-                Exponent: 65537 (0x10001)
-        X509v3 extensions:
-            X509v3 Subject Key Identifier: 
-                D9:FE:21:40:6E:94:9E:BC:9B:3D:9C:7D:98:20:19:E5:8C:30:62:B2
-            X509v3 Authority Key Identifier: 
-                keyid:D9:FE:21:40:6E:94:9E:BC:9B:3D:9C:7D:98:20:19:E5:8C:30:62:B2
-
-            X509v3 Basic Constraints: critical
-                CA:TRUE
-            X509v3 Key Usage: critical
-                Digital Signature, Certificate Sign, CRL Sign
-    Signature Algorithm: sha256WithRSAEncryption
-         9e:45:9e:0c:3b:b6:ef:e1:3a:c8:7c:d1:00:3d:cf:e2:ea:06:
-         b5:b2:3a:bb:06:4b:68:7a:d0:23:97:74:a7:2c:f0:08:d8:79:
-         5a:d7:5a:84:8a:d8:12:9a:1b:d9:7d:5c:4d:70:c5:a5:f9:ab:
-         e5:a3:89:89:dd:01:fa:ec:dd:f9:e9:92:97:db:b0:46:42:f3:
-         d3:62:aa:95:fe:31:67:14:69:58:90:0a:aa:0b:ee:37:23:c7:
-         50:51:b4:f5:7e:9e:e3:7b:f7:e4:cc:42:32:2d:49:0c:cb:ff:
-         49:0c:9b:1e:34:fd:6e:6e:96:8a:79:03:b6:6f:db:09:cb:fd:
-         5f:65:14:37:e1:38:f5:f3:61:16:58:e4:b5:6d:0d:0b:04:1b:
-         3f:50:2d:7f:b3:c7:7a:1a:16:80:60:f8:8a:1f:e9:1b:2a:c6:
-         f9:ba:01:1a:69:bf:d2:58:c7:54:57:08:8f:e1:39:60:77:4b:
-         ac:59:84:1a:88:f1:dd:cb:4f:78:d7:e7:e1:33:2d:fc:ee:41:
-         fa:20:b0:be:cb:f7:38:94:c0:e1:d0:85:0f:bb:ed:2c:73:ab:
-         ed:fe:92:76:1a:64:7f:5b:0d:33:09:07:33:7b:06:3f:11:a4:
-         5c:70:3c:85:c0:cf:e3:90:a8:83:77:fa:db:e6:c5:8c:68:67:
-         10:67:a5:52:2d:f0:c4:99:8f:7f:bf:d1:6b:e2:b5:47:d6:d9:
-         d0:85:99:4d:94:9b:0f:4b:8d:ee:00:5a:47:1d:11:03:ac:41:
-         18:af:87:b7:6f:0c:3a:8f:ca:cf:dc:03:c1:a2:09:c8:e5:fd:
-         80:5e:c8:60:42:01:1b:1a:53:5a:bb:37:a6:b7:bc:ba:84:e9:
-         1e:6c:1a:d4:64:da:d4:43:fe:93:8b:4b:f2:2c:79:16:10:d4:
-         93:0b:88:8f:a1:d8:86:14:46:91:47:9b:28:24:ef:57:52:4e:
-         5c:42:9c:aa:f7:49:ec:27:e8:40:1e:b3:a6:89:22:72:9c:f5:
-         0d:33:b4:58:a3:30:3b:dd:d4:6a:54:93:be:1a:4d:f3:93:94:
-         f7:fc:84:0b:3f:84:20:5c:34:03:44:c5:da:ad:bc:0a:c1:02:
-         cf:1e:e5:94:d9:f3:8e:5b:d8:4c:f0:9d:ec:61:17:bb:14:32:
-         54:0c:02:29:93:1e:92:86:f6:7f:ef:e7:92:05:0e:59:dd:99:
-         08:2e:2e:fa:9c:00:52:d3:c5:66:29:e4:a7:97:44:a4:0e:28:
-         81:13:35:c5:f6:6f:64:e6:41:c4:d5:2f:cc:34:45:25:cf:41:
-         00:96:3d:4a:2e:c2:96:98:4f:4e:4a:9c:97:b7:db:1f:92:32:
-         c8:ff:0f:51:6e:d6:ec:09
-SHA1 Fingerprint=B8:BE:6D:CB:56:F1:55:B9:63:D4:12:CA:4E:06:34:C7:94:B2:1C:C0
-- 
2.41.0.255.g8b1d071c50-goog

